torch/_dynamo/symbolic_convert.py

```diff
@@ -2165,6 +2165,9 @@ def check_inlineable(func):
         if func.get_name() == "patched_init":
             unimplemented("Patched init cannot be inlined.")
 
+        if func.get_name() == "__torch_function__":
+            return
```

Comment:
This seems a bit early, this would make it impossible to disallow `__torch_function__` which seems unexpected.

Reply:
Moved it below the disallow check

Reply:
But you... didn't?

Reply:
So I remember now, I tried this. I can fix this after Yanbo cleans up disallowed/inlining. The exact situation is that I need to disallow torch._tensor._convert, but that means it is not able to be inlined above, which is why this needs to be before the check. In the current system we're missing a "trace through a function in the torch.* namespace without making it a TorchVariable"

Reply:
cc @yanboliang

Reply:
After https://github.com/pytorch/pytorch/pull/109567 we will have a ```FILENAME_ALLOWLIST``` and ```SUBMODULE_ALLOWLIST```, that force to inline the functions in these files or under these modules even the ```torch``` namespace has been skipped.

Reply:
I want to be clear tho that I wouldn't want the resulting function/module to be a TorchVariable though, I want it to be treated like regular user code, a function in this case. Do your changes cover that?